### DISCUSSION TITLE
React 18 support

### DISCUSSION THREAD

**Discussion by stanislavSoro** [xyflow/xyflow#2105] [General]
Hello, when it's planned to support React 18?

**Comment by jonvarfich** [comment]
I am on the same boat. Any news when are you going to support React 18?

**Reply by moklick** [reply]
Hey. We already have a branch for that #2067. We will probably release it next Monday. You can test it by installing `react-flow-renderer@next`. Please let us know if you find any bugs.

**Reply by jonvarfich** [reply]
Now I feel dumb, I should have checked the branches. I will try it out in a bit! Thanks!
